ingestion/src/metadata/ingestion/source/amundsen.py

```diff
@@ -162,7 +163,7 @@ def create_table_entity(self, table):
                 col = Column(**parsed_string)
                 columns.append(col)
 
-            fqn = f"{service_name}{FQDN_SEPARATOR}{database.name}{FQDN_SEPARATOR}{table['schema']}{FQDN_SEPARATOR}{table['name']}"
+            fqn = get_fqdn(service_name, database.name, table["schema"], table["name"])
             table_entity = Table(
```

Comment:
AFAIK we don't need to create the FQDN in the sources, as they get generated on the backend side. Let's just not inform it here

Reply:
skip FQDN. Also might be a good time to change Table to CreateTableRequest here